README.md

```diff
@@ -0,0 +1,20 @@
+# [CollabCamp website](http://collab.camp/)
+
+using [Jekyll](http://jekyllrb.com/)
+
+CollabCamp events bring together social entrepreneurs, sharers, hackers, government officials and citizens to collaboratively work on projects which benefit the collaborative economy (sharing). CollabCamp events are free to attend. We celebrate collaboration and sharing. Let's get it done! 
+
+### Editing this site locally
+
+To clone and run a server locally for testing:
+
+    git clone https://github.com/collabcamp/collabcamp.github.io.git
+    cd collabcamp.github.io
+    sudo npm install jekyll
```

Comment:
@wvengen actually, gem install jekyll fails, attempting to write to:
/var/lib/gems/1.9.1

So, that probably needs a sudo (or arg to install more locally)
Or, we can use npm - which seems to intall locally by default?